app/controllers/guest_rooms_controller.rb

```diff
@@ -0,0 +1,15 @@
+class GuestRoomsController < ApplicationController
+
+  # def create
+  #   GuestRoom.create!(room_id: params[:room_id], guest_id: params[:id])
+  #   redirect_to "/guests/#{params[:id]}"
+  # end
+
+  def create
+    gr = GuestRoom.new(room_id: params[:room_id], guest_id: params[:id])
+      if !gr.save
+        flash[:alert] = gr.errors.full_messages.join(', ')
```

Comment:
Just so you're aware, the user story mentions this not being necessary for the IC - and in the future we won't ask you to implement a sad path in an IC or final. But, this is good practice!